pages/index.html

```diff
@@ -109,17 +109,13 @@
     </p>
   </div>
 
-  <H1>{% trans "And It Just Keeps Getting Better" %}</H1>
-  <p>
-    {% blocktrans %}Each year, a community of DJs, programmers, and artists contribute dozens of new features to Mixxx. <br> <b>And it doesn't cost you a dime.</b>{% endblocktrans %}
-  </p>
-
+  <h1>{% trans "Friendly Community" %}</h1>
   <p>
-    {% blocktrans %}Because Mixxx is <b>open source</b>, anyone can remix or add new features to it. And that includes you!{% endblocktrans %}
+    {% blocktrans %}Mixxx is developed by an international community of DJs, programmers, and artists. Because Mixxx is <b>free (as in freedom and price), open source</b> software, anyone can contribute to make Mixxx even better&mdash;including you!{% endblocktrans %}
   </p>
 
   <p>
-    {% blocktrans with create_a_branch_link="/get-involved#developers" help_with_translations_link="/get-involved#translators" starter_tasks_link="/get-involved" %}You can get involved with Mixxx today by <a href="{{ create_a_branch_link }}">creating your own branch</a>, <a href="{{ help_with_translations_link }}">helping with translations</a>, or working on one of our other <a href="{{ starter_tasks_link }}">starter tasks</a>.{% endblocktrans %}
+    {% blocktrans with create_a_branch_link="/get-involved#developers" help_with_translations_link="/get-involved#translators" starter_tasks_link="/get-involved" mapping_link="/wiki/doku.php/start#controller_mapping_documentation" %}You can get involved with Mixxx today by <a href="{{ mapping_link }}">making a controller mapping</a>, <a href="{{ create_a_branch_link }}">adding features &amp; fixing bugs</a>, <a href="{{ help_with_translations_link }}">helping with translations</a>, or working on one of our other <a href="{{ starter_tasks_link }}">starter tasks</a>.{% endblocktrans %}
```

Comment:
When we release the next beta, I think that would be good to put here. For now, I think it's better without that. I added a link to Launchpad.